pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyLanguageDao.java

```diff
@@ -43,6 +43,10 @@ default long insert(String studyGuid, String languageCode, String name) {
         return getStudyLanguageSql().insert(studyDto.getId(), languageCodeId, name);
     }
 
+    default void update(long umbrellaStudyId, long languageCodeId, String name) {
+        DBUtils.checkUpdate(1, getStudyLanguageSql().update(umbrellaStudyId, languageCodeId, name));
```

Comment:
These changes allow updating study languages in-place. See `UpdateStudyLanguages` task in study-builder.